src/tree/azure/AzureResourceTreeDataProvider.ts

```diff
@@ -107,17 +110,6 @@ export class AzureResourceTreeDataProvider extends AzureResourceTreeDataProvider
                                 iconPath: new vscode.ThemeIcon('mortar-board')
                             }),
                     ];
-                } else {
```

Comment:
I liked having the loading~spin icon while the logging in was happening.

It can be confusing clicking that and it looks like nothing has changed. I've had times where I forgot that I had the "sign in with Azure" webpage opened, but that loading spinner reminded me to go to my browser.

Reply:
Turns out to be _much_ more complicated...

Long story short, if you abandon a sign-in process without doing the browser part, `subscriptionProvider.isSignedIn()` will _block_. So the tree view will spin indefinitely since `getChildren()` cannot return until `isSignedIn()` unblocks--it's not possible to put a spinner _in_ the tree view.

Reply:
Fixed in #728